_docs/services/relational-database.md

```diff
@@ -84,6 +73,35 @@ For example, if you wanted to create a new micro PostgreSQL instance called `my-
 cf create-service aws-rds micro-psql my-service-db
 ```
 
+### Setting optional parameters
+
+There are several optional parameters that you can specify when creating a new instance:
+
+Name               | Description                                                    | Default |
+---                | ---                                                            | ---     |
+`storage`          | Number of GB available to the database instance                | 10      |
+`enable_functions` | Boolean to enable functions on supported databases             | false   |
+`version`          | Specifies a supported major version in RDS (must be in quotes) | ""      |
```

Comment:
maybe instead of `""` we should refer them to the table above? or say aws latest?

Reply:
Good call!

Reply:
Updated this to match the table above.